webextensions/manifest/theme.json

```diff
@@ -464,7 +464,9 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": "62"
+                  "version_added": "62",
+                  "version_removed": "89",
```

Comment:
@rpl is this correct? Given that it's deprecated, presumably, it's still available and hasn't been removed yet, So shouldn't it be flagged with:
```
          "status": {
            "deprecated": true,
          },
```

Reply:
@rebloor I was honestly also considering that, the reason why I went for version_removed is that these colors are going to be ignored and not used anywhere anymore since the Proton restyling, and so they basicallly behave more like a theme property that we never supported (e.g. if chrome adds support for a new theme property which we don't support, then it is also going to be similarly ignore and not be used anywhere), whereas for a deprecated property I'd expect it to still be used but produce a deprecation warning (e.g. if we would be renaming a property and keeping support for the old name for a while as a deprecated way to configure that same theme property).

@rebloor Let me know how this rationale sounds to you?

@Rob--W I'd also like to gather your perspective on this rationale for using version_removed instead of deprecated.

Reply:
Even though there are some references to it in Firefox's source code, it is effectively unused, without doing anything.

There is one difference in comparison with a truly removed property: The `theme.update` method throws for unrecognized properties, whereas `theme.update` does not do anything special for this property.

I think that besides the documentation, the least that we can do is to add `deprecated` to https://searchfox.org/mozilla-central/rev/1aaacaeb4fa3aca6837ecc157e43e947229ba8ce/toolkit/components/extensions/schemas/theme.json#127-130.

I'm voting for `deprecated` due to the observable effect on `theme.update`